docs/UserGuide.md

```diff
@@ -400,34 +400,44 @@ Examples:
 There are a variety of `list` commands that list different types of data.
 
 #### Listing all people 
+If you want to view information of all people stored in VirusTracker, you can use the following command.
 Format: `list l/people`
 
-* Updates the persons list to display all people currently stored in VirusTracker.
+##### What it does
+* It updates the persons list to display all people currently stored in VirusTracker.
 
 #### Listing all infected people 
+If you want to view a list of all infected people, you can use the following command.
 Format: `list l/infected`
 
-* Filters the persons list to display all people that are currently infected.
+##### What it does
+* It filters the persons list to display all people that are currently infected.
 
 #### Listing all quarantined people 
+It you want to view a list of all quarantined people, you can use the following command.
 Format: `list l/quarantined`
 
-* Filters the persons list to display all people currently in quarantine.
+##### What it does
+* It filters the persons list to display all people currently in quarantine.
 
 #### Listing all locations
+It you want ot view information of all locations stored in VirusTracker, you can use the following command.
 Format: `list l/locations`
 
-* Updates the locations list to displays all locations currently stored in VirusTracker.
+##### What it does
+* It updates the locations list to displays all locations currently stored in VirusTracker.
 
 #### Listing all visits
+It you want to view information of all visis stored in VirusTracker, you can use the following command.
 Format: `list l/visits`
 
-* Updates the visits list to displays all visits currently stored in VirusTracker.
+##### What it does
+* It updates the visits list to displays all visits currently stored in VirusTracker.
 
 <div style="page-break-after: always;"></div>
 
 #### Listing high risk locations
-
+If you want to view a list of high risk locations, you can use the following command.
 Lists the locations with high risk of Covid infection.
```

Comment:
Maybe remove this line, if it is already included in the line above?

Reply:
Fixed.